docs/admin.rst

```diff
@@ -294,6 +294,36 @@ web server to apply the changes:
 
   sudo service apache2 restart
 
+.. _submission-cleanup:
+
+Cleaning up deleted submissions
+&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&
+
+When submissions are deleted through the web interface, their database
+records are deleted and their files are securely wiped. For large
```

Comment:
I would say "encrypted files" at least once here to make it clear that we're consistently talking about encrypted data.

Reply:
Good point. I went with twice. :slightly_smiling_face: